images added for yacht section

dmytro-prontenko opened the pull request:
Додав картинки для своєї секції, поки без коду

Commit:
images added for yacht section

Додав картинки для своєї секції

Gunnar97 reviewed (changes requested):
img - добре

в розмітці потрібно вказати розміри та alt
а також у мене не відображалась картинка доки не змінив шлях на цей 
( ../img/yachts-block/mobile/yacht-1@1x-mob.webp)
завтра потрібно буде уточнити я недуже зрозумів з шляхами

Commit:
layout + styles + section img + main.js

Додана розмітка секції, трішки стилів, доданий лінк js і внесений скрипт в main.js

Commit:
layout fixed ul-li instead div

dmytro-prontenko commented:
Ще переробив секцію на ul замість div, не побачив, що це має бути список
